### PR TITLE
Cache 1153 blocks

### PR DESCRIPTION
**Pull request by zancas**

### CODE REVIEW

**Review by Oscar-Pepper — DISMISSED** [review]
wonderful job matey!

**Comment by Oscar-Pepper** [comment]
approved but failed clippy CI. after you have fixed clippy its ready to go!

File: regtest/data/chain_cache/blocks_1153/zcashd/regtest/blocks/index/LOG
Comment (AloeareV) [inline regtest/data/chain_cache/blocks_1153/zcashd/regtest/blocks/index/LOG:1]: Do we need to save log files?

File: regtest/data/chain_cache/blocks_1153/zcashd/regtest/blocks/index/LOG.old
Comment (AloeareV) [inline regtest/data/chain_cache/blocks_1153/zcashd/regtest/blocks/index/LOG.old:1]: As above

File: regtest/data/chain_cache/blocks_1153/zcashd/regtest/chainstate/LOG
Comment (AloeareV) [inline regtest/data/chain_cache/blocks_1153/zcashd/regtest/chainstate/LOG:1]: More log files

File: regtest/data/chain_cache/blocks_1153/zcashd/regtest/chainstate/LOG.old
Comment (AloeareV) [inline regtest/data/chain_cache/blocks_1153/zcashd/regtest/chainstate/LOG.old:1]: log

**Review by AloeareV — CHANGES_REQUESTED** [review]
There are some log files that were committed, which I don't think we need, and if we don't need them they'll potentially confuse someone as to why they're committed! Other than that, looks great

**Review by AloeareV — APPROVED** [review]

**Pull request merged**